src/aa_api_wrapper/main.py

```diff
@@ -1,23 +1,42 @@
 from fastapi import FastAPI, Request
 from dotenv import load_dotenv
 import os
+import uvicorn
+
+from aa_api_wrapper.handlers import (
+    chat_completions_handler,
+    completions_handler,
+    embeddings_handler,
+)
 
-from .handlers import chat_completions_handler, completions_handler, embeddings_handler
 
 app = FastAPI()
 
 load_dotenv()
 
-ALEPH_ALPHA_API_BASE = os.environ.get("ALEPH_ALPHA_API_BASE", "https://api.aleph-alpha.com")
+ALEPH_ALPHA_API_BASE = os.environ.get(
+    "ALEPH_ALPHA_API_BASE", "https://api.aleph-alpha.com"
+)
+
 
 @app.post("/v1/chat/completions")
 async def chat_completions(request: Request):
     return await chat_completions_handler(request)
 
+
 @app.post("/v1/completions")
 async def completions(request: Request):
     return await completions_handler(request)
 
+
 @app.post("/v1/embeddings")
 async def embeddings(request: Request):
     return await embeddings_handler(request)
+
+
+def main():
+    uvicorn.run(app, host="127.0.0.1", port=8000)
```

Comment:
We should probably set this in an .env file, or at least make it variable, s.t. the user has some freedom to set this.